libcxx/test/std/language.support/support.dynamic/new.delete/new.delete.array/new.size_align.replace.pass.cpp

```diff
@@ -27,16 +27,28 @@
 #include "test_macros.h"
 #include "../types.h"
 
+class Data {
+public:
+  Data()  = default;
+  ~Data() = default;
+
+  char* getDummyData() { return this->dummy_data_; }
+
+  std::size_t getDummyDataSize() { return sizeof(this->dummy_data_); }
+
+private:
+  alignas(OverAligned) char dummy_data_[alignof(OverAligned) * 3];
```

Comment:
I don't understand why indirecting through a class solves anything here. After all you're still returning the address of a global variable.

Reply:
I have removed the classes and reverted to the initial behaviour. Please see the initial comment as the changes made are different to those previously reviewed. Thanks